app/position/abstractpositionprovider.h

```diff
@@ -26,10 +26,14 @@ class GeoPosition : public QgsGpsInformation
 
     double verticalSpeed = -1;
 
+    QString fixStatusString;
```

Comment:
If it is class, why we do not use private members and setters/getters? I am not big fan of public members in general :)

Reply:
Well yeah, it's used like this in `QgsGpsInformation` (base class of this), look: https://github.com/qgis/QGIS/blob/master/src/core/gps/qgsgpsconnection.h#L98 I only added few other members